src/lein_monolith/task/each.clj

```diff
@@ -137,7 +217,25 @@
                         :elapsed (/ (- (System/nanoTime) start) 1000000.0)})]
     (try
       (lein/info (format "\nApplying to %s" (ansi/sgr target :bold :yellow)))
-      (apply-subproject-task (:monolith ctx) subproject (:task ctx))
+      (if-let [out-dir (get-in ctx [:opts :output] )]
+        ; Capture output to file.
+        (let [out-file (io/file out-dir (:group subproject) (str (:name subproject) ".txt"))]
```

Comment:
The `:group` and `:name` fields already correspond to the namespace and name of the project's symbol. Are you asking what happens if a project does something like `(defproject ~(symbol "foo" "bar/baz") ...`? In that (illegal) case I think it would create an additional level of nesting.